src/components/Column/Column.test.js

```diff
@@ -18,7 +28,12 @@ function renderColumn(args) {
 
 it('should render the column', () => {
   const { getByText } = renderColumn();
-  getByText('Went well');
+  getByText('Any title');
```

Comment:
This tests are all dependent on the setup you have above.  You can change the props dynamically to each test and have all you need on the test and not in one function. For example: 

```
it('should render the column', () => {
  const title = 'Any title';
  const { getByText } = renderColumn({ title });
  getByText(title);
});
```

With this structure, everything you are testing is in the test and not on a single function.

Reply:
done as suggested